common/containers/Tabs/SendTransaction/components/UnitDropdown.jsx

```diff
@@ -1,87 +1,40 @@
 // @flow
 import React from 'react';
+import SimpleDropDown from 'components/ui/SimpleDropdown';
 
-class Option extends React.Component {
-  props: {
-    value: string,
-    active: boolean,
-    onChange: (value: string) => void
-  };
-  render() {
-    const { value, active } = this.props;
-    return (
-      <li>
-        <a className={active ? 'active' : ''} onClick={this.onChange}>
-          {value}
-        </a>
-      </li>
-    );
-  }
-
-  onChange = () => {
-    this.props.onChange(this.props.value);
-  };
-}
+type UnitDropdownProps = {
+  value: string,
+  options: string[],
+  onChange?: (value: string) => void
```

Comment:
Should this be a type of synthetic event?

Reply:
@wbobeirne Mentioned he would prefer that it is not.

Reply:
^ Basically, dropdowns just receive click events, they're not input changes, so an event doesn't have a `value` attribute. In general, I think components should abstract out their markup as much as possible.